Pocket/src/main/java/com/pocket/app/reader/DestinationHelper.kt

```diff
@@ -43,35 +42,37 @@ class DestinationHelper @Inject constructor(
             forceOpenInWebView -> {
                 Destination.ORIGINAL_WEB
             }
-            // is video
+            // video
             item?.type == ItemType.VIDEO -> {
                 Destination.ORIGINAL_WEB
             }
-            // is collection
-            httpUrl.host == "getpocket.com"
+            // collection
+            isHostedByPocket(httpUrl)
                     && httpUrl.pathSegments.size >= 2
                     && httpUrl.pathSegments[0] == "collections" -> {
                 Destination.COLLECTION
             }
+            // german collection
+            isHostedByPocket(httpUrl)
+                    && httpUrl.pathSegments.size == 3
+                    && httpUrl.pathSegments[0] == "de"
```

Comment:
This should support any locale https://getpocket.com/fr-FR/collections/pockets-top-saved-stories-of-the-week-november-9-2024 is a valid collection url